canary/scripts/update_packages.sh

```diff
@@ -0,0 +1,10 @@
+set -e
+
+VERSION=$(npx semver $1)
```

Comment:
Should we validate this is a valid `VERSION`, as we do here?

https://github.com/AztecProtocol/aztec-packages/blob/cd78ec9afa887b1e9ac0b446b110603fad29e7e2/build-system/scripts/deploy_npm#L16-L21

Reply:
Done